libmscore/textbase.cpp

```diff
@@ -10,6 +10,8 @@
 //  the file LICENCE.GPL
 //=============================================================================
 
+#include "global/log.h"
```

Comment:
I suggest to not include it here, otherwise we can easily miss this during the port of the changes

in short, we're going to reduce external dependencies from "libmscore" in master

Reply:
OK, removed this. Reduces this PR to a port of @mattmcclinch's fix for master to 3.x